geode-core/src/main/java/org/apache/geode/management/internal/configuration/mutators/ConfigurationManager.java

```diff
@@ -27,9 +27,9 @@
 @Experimental
 public interface ConfigurationManager<T extends AbstractConfiguration> {
 
-  void add(T config, String groupName);
+  void add(T config, String groupName) throws Exception;
```

Comment:
Could we make this more specific instead of "throws Exception"?

Reply:
as commented before.